packages/styles/scss/components/pagination-nav/_pagination-nav.scss

```diff
@@ -0,0 +1,197 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../type' as *;
+@use '../../theme' as *;
+@use '../../utilities/button-reset';
+@use '../../utilities/convert' as *;
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/high-contrast-mode' as *;
+@use '../../utilities/visually-hidden' as *;
+
+/// Pseudo underline
+/// @access public
+/// @group pagination-nav
+@mixin pseudo-underline() {
+  &:not(.#{$prefix}--pagination-nav__page--direction) {
+    &::after {
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      display: block;
+      width: 0;
+      height: $spacing-02;
+      background-color: $background-brand;
+      content: '';
+      opacity: 0;
+      transition: width $duration-fast-02 motion(standard, productive);
+    }
+  }
+
+  .#{$prefix}--pagination-nav__page--active + &::after,
+  &.#{$prefix}--pagination-nav__page--active::after {
+    left: calc(50% - #{$spacing-05 / 2});
+    width: $spacing-05;
+    opacity: 1;
+  }
+}
```

Comment:
Would it make sense to have this in its own `mixins.scss` file?

Reply:
The current `pagination-nav` doesn't have one... 🤷🏽‍♀️

Reply:
just updated it with a new `_mixins` file :)